decidim-assemblies/spec/system/admin/valuator_checks_components_spec.rb

```diff
@@ -49,4 +47,12 @@
       end
     end
   end
+
+  context "when the user has not accepted the admin TOS" do
+    let!(:user) { create :user, :confirmed, organization: }
+
+    it "shows a message to accept the admin TOS" do
+      expect(page).to have_content("Please take a moment to review the admin terms of service")
+    end
+  end
```

Comment:
Since this part is pretty similar for multiple specs, maybe we should extract it to a shared example?